packages/utilities/label.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from "react";
+import { InputAppearance } from "../shared/types/inputAppearance";
+import { cx } from "emotion";
+import {
+  tintText,
+  visuallyHidden,
+  flex,
+  margin
+} from "../shared/styles/styleUtils";
+import { getLabelStyle } from "../shared/styles/formStyles";
+import {
+  themeError,
+  greyLightDarken2,
+  iconSizeXs
+} from "../design-tokens/build/js/designTokens";
+import Tooltip from "../tooltip/components/Tooltip";
+import { Icon } from "../icon";
+import { SystemIcons as SI } from "../icons/dist/system-icons-enum";
+
+const defaultTrigger = (
+  <Icon color={greyLightDarken2} shape={SI.CircleQuestion} size={iconSizeXs} />
+);
+
+export const renderLabel: React.FC<{
+  appearance?: string;
+  hidden?: boolean;
+  id: string;
+  label?: React.ReactNode;
+  required?: boolean;
+  tooltipContent?: React.ReactNode;
+  tooltipTrigger?: React.ReactNode;
+}> = ({
+  appearance,
+  hidden = false,
+  id,
+  label,
+  required = false,
+  tooltipContent,
+  tooltipTrigger = defaultTrigger
+}) => {
+  const hasError = appearance === InputAppearance.Error;
+  const labelClassName = hidden ? cx(visuallyHidden) : getLabelStyle(hasError);
+  const labelNode = (
+    <label className={labelClassName} htmlFor={id} data-cy="textInput-label">
+      {label}
+      {required ? <span className={cx(tintText(themeError))}> *</span> : null}
+    </label>
+  );
+  if (!tooltipContent) {
+    return labelNode;
+  }
+  return (
+    <div className={flex({ align: "center" })}>
```

Comment:
i wonder why we need `align: "center"` in this use case it does not seem to be necessary.

Reply:
I don't remember why this was added, but removing it doesn't seem to have any affect. I'm fine with removing it :)